canbench-bin/src/main.rs

```diff
@@ -53,6 +61,8 @@ fn main() {
             .unwrap_or(&DEFAULT_RESULTS_FILE.to_string()),
     );
 
+    let custom_drun_path = cfg.get("drun_path").map(PathBuf::from);
```

Comment:
I'm not a fan of adding `drun_path` to the configuration. The fact that we're using `drun` is an implementation detail, and we already have plans to swap out `drun` in favor of `PockerIC`, so adding it explicitly would expose it in the interface. Is this a necessary feature to add for bazel to work?

Reply:
I see. It does impose some restrictions on using canbench as any `bazel build` step (AFAIK there is no network access during any `build`), but it seems OK just to run a benchmark using `bazel run`. Considering drun is being replaced it seems OK to revert the drun related changes for now.

Reply:
I see. It does impose some restrictions on using canbench as any `bazel build` step (AFAIK there is no network access during any `build`), but it seems OK just to run a benchmark using `bazel run`. Considering drun is being replaced it seems OK to revert the drun related changes for now.